### ISSUE TITLE
Sensitivity applied before deadzone for dual axis inputs

### ISSUE THREAD

**Issue by yrns** [Leafwing-Studios/leafwing-input-manager#495]
## Version

0.13.3

## Operating system & version

NixOS unstable

## What you did

Set input sensitivity <= deadzone, i.e.: 

```
DualAxis::right_stick()
            .with_sensitivity(0.1, 0.1)
            .with_deadzone(DeadZoneShape::Ellipse {
                radius_x: 0.1,
                radius_y: 0.1,
            }),
```

## What you expected to happen

Non-zero input values.

## What actually happened

Zeroes.

## Additional information

If the sensitivity for a dual axis input is less than or equal to the thresholds, the axis will never be considered "pressed". It seems `extract_dual_axis_data` ignores the deadzone for each axis (as it should) but applies the sensitivity, then checks the deadzone later. In the aforementioned case the inputs are never outside the deadzone. I think this is a bug. The sensitivity should be applied after the deadzone is checked?

**Comment by Shute052** [comment]
should this be fixed by #491?

**Comment by yrns** [comment]
> it looks like #494 should have fixed this

I will check when merged if this is still open.
